### PR TITLE
Fixes odd looking css: in image module everything should be in one row

### PR DESCRIPTION
**Pull request by Sagarpreet**
Before: 

Screenshot 2020-10-07 at 1 44 58 PM

After:

Screenshot 2020-10-07 at 2 04 06 PM

### CODE REVIEW

File: dist/PublicLab.Editor.js [hunk 1]
```diff
@@ -370,7 +369,7 @@ function create (options) {
     bureaucrat.emit('valid', validFiles);
     var form = new FormData();
     Object.keys(o.formData).forEach(function copyFormData(key) {
-      form[key] = o.formData[key];
+      form.append(key, o.formData[key]);
```
Comment (jywarren) [inline dist/PublicLab.Editor.js:372]: Ah, was this change from recent uncompiled changes or is it accidentally in dist? Thanks!

File: .gitignore [hunk 2]
```diff
@@ -40,3 +40,6 @@ todo.txt
 
 # Test output file
 _SpecRunner.html
+
+.DS_Store
```
Comment (jywarren) [inline .gitignore:44]: Nice.

File: dist/PublicLab.Editor.css [hunk 3]
```diff
@@ -150,6 +150,7 @@
 .ple-module {
   padding: 20px;
   border-top: 1px solid #ddd;
+  display: flex;
```
Comment (jywarren) [inline dist/PublicLab.Editor.css:153]: Great!

**Comment by Sagarpreet** [comment]
```
  ReferenceError: Can't find variable: PL in file:///home/travis/build/publiclab/PublicLab.Editor/spec/javascripts/button_spec.js (line 7) (1)
     Expected false to be true. (2)
 Editor
   - exists, and has a textarea...X exists, and has a textarea
     ReferenceError: Can't find variable: PL in file:///home/travis/build/publiclab/PublicLab.Editor/spec/javascripts/editor_spec.js (line 7) (1)
     Expected undefined not to be undefined. (2)
     TypeError: undefined is not an object (evaluating 'editor.options') in file:///home/travis/build/publiclab/PublicLab.Editor/spec/javascripts/editor_spec.js (line 16) (3)
   - counts valid modules and enables publish button...X counts valid modules and enables publish button
     ReferenceError: Can't find variable: PL in file:///home/travis/build/publiclab/PublicLab.Editor/spec/javascripts/editor_spec.js (line 7) (1)
     TypeError: undefined is not an object (evaluating 'editor.titleModule') in file:///home/travis/build/publiclab/PublicLab.Editor/spec/javascripts/editor_spec.js (line 22) (2)
   - sends AJAX request on editor.publish()...X sends AJAX request on editor.publish()
     ReferenceError: Can't find variable: PL in file:///home/travis/build/publiclab/PublicLab.Editor/spec/javascripts/editor_spec.js (line 7) (1)
     TypeError: undefined is not an object (evaluating 'editor.options') in file:///home/travis/build/publiclab/PublicLab.Editor/spec/javascripts/editor_spec.js (line 41) (2)
 Errors
   - displays given errors...X displays given errors
     ReferenceError: Can't find variable: PL in file:///home/travis/build/publiclab/PublicLab.Editor/spec/javascripts/errors_spec.js (line 7) (1)
   - does not display error alert if there are no errors...X does not display error alert if there are no errors
     ReferenceError: Can't find variable: PL in file:///home/travis/build/publiclab/PublicLab.Editor/spec/javascripts/errors_spec.js (line 25) (1)
 Formatter
```

**Comment by jywarren** [comment]
This looks like a syntax error somewhere, causing the whole PL object to not get defined properly. Could it be, somehow? Or might it have failed to fetch a dependency and then caused that error in compilation?

**Comment by sagarpreet-chadha** [comment]
It has to be unrelated to this PR as it has only minor CSS change. What do you think?

**Comment by jywarren** [comment]
Very strange. Maybe try rebasing it?

**Comment by Sagarpreet** [comment]
@jywarren after rebasing, tests pass 🎉

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
Super!!!

**Pull request merged**